Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -785,7 +790,6 @@ void HTMLInputElement::attributeChanged(const QualifiedName& name, const AtomStr
         return;
 
     ASSERT(m_inputType);
-    Ref protectedInputType { *m_inputType };
```

Comment:
Seems the previous code was better since it was avoiding repeated calls to `protectedInputType()` and thus there was less ref-counting churn. Though in modern code we would name this variable `inputType` instead of `protectedInputType`